src/engine/Match_rules.ml

```diff
@@ -100,7 +100,7 @@ let group_rules xconf rules xtarget =
            | _ when not relevant_rule -> Right3 r
            | `Taint _ as mode -> Left3 { r with mode }
            | (`Extract _ | `Search _) as mode -> Middle3 { r with mode }
-           | `Step _ ->
+           | (`Step _ | `Secrets _)->
```

Comment:
so I guess this will be handled at some point in semgrep-pro?

Reply:
Yes; evaluation happens there.